exec/java-exec/src/main/resources/rest/profile/list.ftl

```diff
@@ -154,15 +154,19 @@
       <strong>No running queries.</strong>
     </div>
   </#if>
+
+  <#include "*/alertModals.ftl">
+
   <table width="100%">
     <script type="text/javascript" language="javascript">
     //Validate that the fetch number is valid
     function checkMaxFetch() {
       var maxFetch = document.forms["profileFetch"]["max"].value;
       console.log("maxFetch: " + maxFetch);
       if (isNaN(maxFetch) || (maxFetch < 1) || (maxFetch > 100000) ) {
-        alert("Invalid Entry: " + maxFetch + "\n" +
-               "Please enter a valid number of profiles to fetch (1 to 100000) ");
+	    populateAndShowAlert('invalidProfileFetchSize',maxFetch);
+        //alert("Invalid Entry: " + maxFetch + "\n" +
```

Comment:
remove commented out alert statements from here and other files

Reply:
👍